aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -364,6 +367,22 @@ bool TuningContext::IsTuningEnabled() const {
   return tuning_enable_;
 }
 
+void TuningContext::EnableNumericsCheck() {
+  numerics_check_enabled_ = true;
+}
+
+void TuningContext::DisableNumericsCheck() {
+  numerics_check_enabled_ = false;
+}
+
+bool TuningContext::IsNumericsCheckEnabled() const {
+  static const char *env = getenv("PYTORCH_TUNABLEOP_NUMERICAL_CHECK");
+  if (env != nullptr && strcmp(env, "0") == 0) {
```

Comment:
Ah, It seems env has a higher priority, and set API has lower priority

Reply:
Correct, env vars have higher priority than the APIs.